client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-info.js

```diff
@@ -258,13 +260,19 @@ const mapStateToProps = ( state, { orderId, siteId } ) => {
 	const shippingLabel = getShippingLabel( state, orderId, siteId );
 	const storeOptions = loaded ? shippingLabel.storeOptions : {};
 	const errors = loaded && getFormErrors( state, orderId, siteId ).packages;
+	const isInternationalOrder =
+		has( shippingLabel, 'form.destination' ) &&
```

Comment:
Nitpick: Using `'object.path.represented.as.string'` is significantly slower in lodash because it has to parse the path using a RegExp. `has( shippingLabel, [ 'form', 'destination' ] )` is better.

Reply:
Cool, thanks for the tip.